modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItSqlApiBaseTest.java

```diff
@@ -0,0 +1,860 @@
+                "Can`t delete column, belongs to primary key: [name=ID]",
+                ses,
+                "ALTER TABLE TEST DROP COLUMN id"
+        );
+
+        // TODO: IGNITE-19150 We are waiting for schema synchronization to avoid races to create and destroy indexes
+        waitForIndexBuild("TEST", "TEST_IDX3");
+        checkDdl(true, ses, "DROP INDEX TESt_iDX3");
+
+        // DROP COLUMNS
+        checkDdl(true, ses, "ALTER TABLE TEST DROP COLUMN VAL1");
+        checkError(
+                TableNotFoundException.class,
+                ErrorGroups.Table.TABLE_NOT_FOUND_ERR,
+                "The table does not exist [name=\"PUBLIC\".\"NOT_EXISTS_TABLE\"]",
+                ses,
+                "ALTER TABLE NOT_EXISTS_TABLE DROP COLUMN VAL1"
+        );
+        checkDdl(false, ses, "ALTER TABLE IF EXISTS NOT_EXISTS_TABLE DROP COLUMN VAL1");
+        checkError(
+                ColumnNotFoundException.class,
+                ErrorGroups.Table.COLUMN_NOT_FOUND_ERR,
+                "Column does not exist [tableName=\"PUBLIC\".\"TEST\", columnName=\"VAL1\"]",
+                ses,
+                "ALTER TABLE TEST DROP COLUMN VAL1"
+        );
+
+        // DROP TABLE
+        checkDdl(false, ses, "DROP TABLE IF EXISTS NOT_EXISTS_TABLE");
+
+        checkDdl(true, ses, "DROP TABLE TEST");
+        checkError(
+                TableNotFoundException.class,
+                ErrorGroups.Table.TABLE_NOT_FOUND_ERR,
+                "The table does not exist [name=\"PUBLIC\".\"TEST\"]",
+                ses,
+                "DROP TABLE TEST"
+        );
+
+        checkDdl(false, ses, "DROP INDEX IF EXISTS TEST_IDX");
+
+        checkError(
+                IndexNotFoundException.class,
+                Index.INDEX_NOT_FOUND_ERR,
+                "Index does not exist [name=\"PUBLIC\".\"TEST_IDX\"]", ses,
+                "DROP INDEX TEST_IDX"
+        );
+    }
+
+    /** Check all transactions are processed correctly even with case of sql Exception raised. */
+    @Test
+    public void implicitTransactionsStates() {
+        IgniteSql sql = igniteSql();
+
+        if (sql instanceof ClientSql) {
+            return;
+        }
+
+        sql("CREATE TABLE TEST(ID INT PRIMARY KEY, VAL0 INT)");
+
+        Session ses = sql.createSession();
+
+        TxManager txManager = txManager();
+
+        for (int i = 0; i < ROW_COUNT; ++i) {
+            assertThrowsSqlException(
+                    Catalog.VALIDATION_ERR,
+                    "Table with name 'PUBLIC.TEST' already exists",
+                    () -> execute(ses, "CREATE TABLE TEST(ID INT PRIMARY KEY, VAL0 INT)")
+            );
+        }
+
+        // No new transactions through ddl.
+        assertEquals(0, txManager.pending());
+    }
+
+    /** Check correctness of implicit and explicit transactions. */
+    @Test
+    public void checkTransactionsWithDml() {
+        IgniteSql sql = igniteSql();
+
+        if (sql instanceof ClientSql) {
```

Comment:
I think if some test cases are not expected to run for sql clients, then it is better to use `Assumptions.assumeFalse(sql instanceof ClientSql)' instead of simple returns.

Reply:
Looks like these test fail with the same error (` Error occurred when notifying safe time advanced callback`), so I think it is better to disable these test cases and provide a like to a ticket.